test/providers/keychain/stop

```diff
@@ -1,28 +1,5 @@
 #!/bin/bash
-
 set -euo pipefail
 
-# load the environment with the test config
-source load_test_env_vars
-
-for (( idx=1; idx<=NUM_SECRETS; idx++ ))
-do
-  secret_account="${ACCOUNT}_${idx}"
-  secret_service="${SERVICE}_${idx}"
-
-  # delete the password if it exists, and send stderr to stdout
-  # do not die if the request errors
-  delete_output="$(security delete-generic-password \
-                  -a "${secret_account}" \
-                  -s "${secret_service}" 2>&1)" || true
-
-  # if output includes "SecKeychainSearchCopyNext", password does not exist
-  # overwrite output messages for clarity / simplicity
-  if [[ "${delete_output}" == *"SecKeychainSearchCopyNext"* ]]; then
-    echo "Password does not exist."
-  elif [[ "${delete_output}" == *"password has been deleted"* ]]; then
-    echo "Password has been deleted."
-  else
-    echo "${delete_output}"
-  fi
-done
+# Clean up
+go run ./...
```

Comment:
Curious, maybe you can help me understand... How does the `stop` script, when run as a standalone process,  have access to deleting the keychain secrets without prompting the user? I thought the keychain only trusts the process that wrote the secrets (i.e. the `test` process)? I know `stop` works standalone, I've tried it.

Reply:
Trust is only necessary for reads.